keycloak/keycloak-ui
# Change 'Resources and Auth Scopes' label

## Motivation
Fixes #4312 

Also adds missing localization key for selectARole.

## Brief Description
When opening the Evaluation Tool for a client with Authorization enabled, the label set to the field related to resources and scopes is `Resources and Authentication Scopes`.

Instead of that, the label should be `Resources and Scopes` giving that scopes here are authorization-related and we can make the label less verbose.

## Verification Steps
1. Enable authorization for a client.
2. Go to Authorization tab
3. Got to Evaluate sub-tab
4. See changed label as specified in #4312
